ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/image/_cq_editConfig.xml

```diff
@@ -23,8 +23,8 @@
             <plugins jcr:primaryType="nt:unstructured">
                 <crop
                     jcr:primaryType="nt:unstructured"
-                    supportedMimeTypes="[image/jpeg,image/png,image/webp,image/tiff]"
-                    features="*">
+                    features="*"
```

Comment:
@DeepaliAgrwl : what are the changes in Image component as part of this PR ?

Reply:
@rishabh-behl  I have reverted back the changes for this xml